app/Services/Pop3/Mailbox.php

```diff
@@ -7,9 +7,9 @@ class Mailbox{
     /** @var \PhpImap\Mailbox mailbox*/
     protected $mailbox;
 
-    function login($host, $port, $user, $pass, $folder = "INBOX", $ssl = false) {
+    function login($host, $port, $user, $pass, $folder = "INBOX", $ssl = false, $options ="/pop3") {
```

Comment:
this will failg when sending null to options, (the default case) so I would do a null check first
```if( $options == null) $options = '/pop3'``` along with the default value in case it is not sent